src/main/java/jackyy/dimensionaledibles/util/TeleporterHandler.java

```diff
@@ -120,20 +120,22 @@ public static void updateDimPos(EntityPlayer player, int dimension, BlockPos pos
     public static BlockPos getDimPos(EntityPlayerMP player, int dimension, BlockPos currentPos) {
         NBTTagCompound dimensionCache = getModNBTData(player);
         NBTTagCompound dimPos = (NBTTagCompound) dimensionCache.getTag("" + dimension);
-        BlockPos position;
         if (dimPos == null) {
             dimPos = new NBTTagCompound();
             dimensionCache.setTag("" + dimension, dimPos);
+        }
+        BlockPos position = new BlockPos(dimPos.getInteger("x"), dimPos.getInteger("y"), dimPos.getInteger("z"));
```

Comment:
Could fetching here cause an issue if the dimension position was just created? If `dimPos` was null, it will be initialized as a new NBTTagCompound, which would not have any stored x/y/z coordinate positions yet

Reply:
If it was just created, then all fields will resolve to the default value of `0`. Since we only care about the Y position being greater than zero, this should work fine.